internal/websocket/ws.go

```diff
@@ -115,15 +122,15 @@ func New(opts ...Option) (*Websocket, error) {
 
 	defaults := []Option{
 		NowFunc(time.Now),
-		FetchURLTimeout(30 * time.Second),
-		PingInterval(30 * time.Second),
-		PingTimeout(90 * time.Second),
-		ConnectTimeout(30 * time.Second),
-		KeepAliveInterval(30 * time.Second),
-		IdleConnTimeout(10 * time.Second),
-		TLSHandshakeTimeout(10 * time.Second),
-		ExpectContinueTimeout(1 * time.Second),
-		MaxMessageBytes(256 * 1024),
+		FetchURLTimeout(fetchUrlTimeoutDefault),
```

Comment:
We may want to simply remove all the default values from here & instead favor moving them into the `cmd/xmidt-agent/config.go` file as built in defaults to the actual configuration.  We will also want to add an `Enabled` flag there as well ... that way through configuration this can be turned on/off & the defaults are expressed in the configuration system & to the end user.